src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -26,16 +20,12 @@ public IntakeSubsystem() {
     // Restore motors to factory defaults for settings to be consistent
     m_intakeRoller.restoreFactoryDefaults();
     m_intakeLift.restoreFactoryDefaults();
-    m_innerRoller.restoreFactoryDefaults();
-
-    // Lower indexer belt & inner roller follow the intake roller
-    m_lowerIndexBelt.follow(m_intakeRoller, true);
-    m_innerRoller.follow(m_intakeRoller, true);
 
     // Lift shouldn't drift, so set it to brake mode
     m_intakeLift.setIdleMode(CANSparkMax.IdleMode.kBrake);
-    // Set max current the winch can draw
-    m_intakeLift.setSmartCurrentLimit(kLiftMaxExpectedCurrent, 0, 20000);
+
+    // Set max current the lift can draw
+    m_intakeLift.setSmartCurrentLimit(kLiftMaxExpectedCurrent);
```

Comment:
I'm not sure why Abiral and I used the manually specified version above. I see that the simplified version has the same arguments, so way to KISS.

Reply:
That reminds me - I don't think we ever set any current limits for the drive motors, which would probably be a good thing to do. I'll make that change real quick :)